tests/acceptance/AbstractContainerTest.php

```diff
@@ -4,7 +4,7 @@
 
 use PHPUnit_Framework_TestCase;
 use tests\support\TestFileCreator;
-use TomPHP\ConfigServiceProvider\ConfigureContainer;
+use TomPHP\ConfigServiceProvider\Configurator;
 
 abstract class AbstractContainerTest extends PHPUnit_Framework_TestCase
```

Comment:
@tomphp 

Similarly, should we rename `PimpleContainerTest` to `PimpleContainerAdapterTest`, etc.?